minigrid/envs/babyai/goto.py

```diff
@@ -11,9 +11,52 @@
 
 class GoToRedBallGrey(RoomGridLevel):
     """
+
+    ## Description
+
     Go to the red ball, single room, with distractors.
     The distractors are all grey to reduce perceptual complexity.
     This level has distractors but doesn't make use of language.
+
+    ## Mission Space
+
+    "go to the red ball"
+
+    ## Action Space
+
+    | Num | Name         | Action            |
+    |-----|--------------|-------------------|
+    | 0   | left         | Turn left         |
+    | 1   | right        | Turn right        |
+    | 2   | forward      | Move forward      |
+    | 3   | pickup       | Pick up an object |
+    | 4   | drop         | Unused            |
+    | 5   | toggle       | Unused            |
+    | 6   | done         | Unused            |
+
+    ## Observation Encoding
+
+    - Each tile is encoded as a 3 dimensional tuple:
+        `(OBJECT_IDX, COLOR_IDX, STATE)`
+    - `OBJECT_TO_IDX` and `COLOR_TO_IDX` mapping can be found in
+        [minigrid/minigrid.py](minigrid/minigrid.py)
+    - `STATE` refers to the door state with 0=open, 1=closed and 2=locked
+
+    ## Rewards
+
+    A reward of '1' is given for success, and '0' for failure.
+
+    ## Termination
+
+    The episode ends if any one of the following conditions is met:
+
+    1. The agent goes to the red ball.
+    2. Timeout (see `max_steps`).
+
+    ## Registered Configurations
+
+    - `BabyAI-GoToRedBallGrey-v0`
```

Comment:
Is it `BabyAI/GoTo...` or `BabyAI-GoTo..` I thought it was the first but I am probably wrong

Reply:
From the registered env names it is `BabyAI-GoTo..`, I also ran the envs, using `BabyAI-GoTo..` works. See https://github.com/Farama-Foundation/Minigrid/blob/da409c4fc6db880613c819ab097d22522e602196/minigrid/__init__.py#L515

Reply:
Ok that is fine